I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/mash_test.rpy", line 213, in script
    "Oh fuck, is that patronizing? He hasn't had a chance to look yet..."
  File "game/screens.rpy", line 423, in execute
    screen countdown:
  File "game/screens.rpy", line 423, in execute
    screen countdown:
  File "game/screens.rpy", line 425, in execute
    if sprite_change == True:
  File "game/screens.rpy", line 425, in <module>
    if sprite_change == True:
NameError: name 'sprite_change' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/mash_test.rpy", line 213, in script
    "Oh fuck, is that patronizing? He hasn't had a chance to look yet..."
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\exports.py", line 1473, in say
    who(what, *args, **kwargs)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\character.py", line 1352, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\character.py", line 1003, in do_display
    display_say(who,
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\character.py", line 693, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 3579, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 4052, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\screen.py", line 476, in visit_all
    callback(self)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 4052, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\screen.py", line 487, in per_interact
    self.update()
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\screen.py", line 680, in update
    self.screen.function(**self.scope)
  File "game/screens.rpy", line 423, in execute
    screen countdown:
  File "game/screens.rpy", line 423, in execute
    screen countdown:
  File "game/screens.rpy", line 425, in execute
    if sprite_change == True:
  File "game/screens.rpy", line 425, in <module>
    if sprite_change == True:
NameError: name 'sprite_change' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.1.23060707
detective test 1.2
Sun Feb 18 15:35:17 2024
